Adapt regex in test to match new timestamp format

wbiker opened the pull request:
Hi, I noticed a test fail and took a look. Looks like the standard format has changed. At least the test fails because there is no longer an Z at the end of the timestamp.

I removed it and hope that fixes this kind of issue for good.

wbiker committed:
Adapt regex in test to match new timestamp format

whity commented:
hi @wbiker , thanks, but i can't see anything failing at the moment

wbiker commented:
Maybe a good idea to remove the standard timestamp test. It does not matter how the timestamp looks like and makes only problems with different system settings.

whity commented:
what about make the `Z` optional `    like($log.output.get, /^ '[' \d+ '-' \d+ '-' \d+ 'T' \d+ ':' \d+ ':' \d+ '.' \d+ 'Z'? ']'/, 'standard timestamp');`?

wbiker commented:
Will modify the code according to your suggestion

wbiker committed:
Make Z in date regex optional

Test for standard timestamp failed on operating systems with not expected
timezone. Made the regex timezone part optional.

whity commented:
hey @wbiker your last change, makes `Z]` optional not only `Z`, any reason why to include the square bracket? :)

wbiker committed:
Just make Z optional

wbiker commented:
Good questions. I recognized the original regex ended with 'Z]'. Thought that makes sense. But after your questions I do not know any reason to include the ].

whity committed:
keep square bracket but make 'Z' optional
